packages/showcases/core/csf/components/actions/dropdownItem.csf.js

```diff
@@ -0,0 +1,47 @@
+export const parameters = {
+  actions: {
+    handles: ['mouseenter', 'click', 'focusin', 'focusout'],
+  },
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/?node-id=2421%3A3',
+  },
+};
+
+export const argTypes = {
+  label: {
```

Comment:
I think `text` should be better here for a single dropdown item.

Reply:
For me, I prefer use a slot for the label 🤔